card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
home view is specific to a given slot (nothing to do with main).

if, for example, you click "cancel" on a given form on a card in a slot, then you go back to that slot's home view.  if it was originally `content`, then its home view is `content`, and you go back to `content` when you click cancel.